mix.exs

```diff
@@ -30,41 +30,44 @@ defmodule Barrel.Mixfile do
         :ssl,
         :os_mon,
         :inets,
+        :p1_utils,
         :gproc,
         :ibrowse,
-        :snappy,
         :jsx,
         :ucol,
         :oauth,
-        :mochiweb,
-        :mochicow,
+        :fast_yaml,
         :ranch,
         :cowboy,
+        :mochiweb,
+        :mochicow,
         :hooks,
         :econfig,
         :hackney,
-        :exometer_core
+        :exometer_core,
+        :snappy
       ]
     ]
   end
 
   defp deps do
     [
-      {:lager, "3.0.2"},
+      {:barrel_ex_logger, git: "https://github.com/barrel-db/barrel_ex_logger.git"},

```

Comment:
How costly is enough compared to using logger directly? If we use logger, how is it configured?

is this easier to use https://github.com/tolbrino/hut/tree/master/examples/elixir ?